src/main/java/lotto/LottoTVShow.java

```diff
@@ -0,0 +1,66 @@
+package lotto;
+
+import lotto.domain.Lotto;
+import lotto.domain.LottoResult;
+import lotto.view.InputView;
+import lotto.view.ResultView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoTVShow {
+	private int purchaseAmount;
+
+	private int lottoTryCount;
+
+	private List<List<Integer>> autoGeneratedLottos = new ArrayList<>();
+
+	private List<Integer> lastWeekPrizeNumbers = new ArrayList<>();
+
+	Lotto lotto = new Lotto();
+
+	LottoResult lottoResult = new LottoResult();
+
+	InputView inputView = new InputView();
+
+	ResultView resultView = new ResultView();
+
+
+	public void broadcast() {
+		printLottoShowIntro();
+
+		printChooseLottoNumber();
+
+		printLastWeekPrizeNumber();
+
+		List<Integer> lottoResultList = compareLastWeekPrizeNumberWithAutoGeneratedLottoNumber();
+
+		lottoResult.arrangePrize(lottoResultList, lotto.LOTTO_PRICE_PER_SHEET);
```

Comment:
```suggestion
		lottoResult.arrangePrize(lottoResultList, Lotto.LOTTO_PRICE_PER_SHEET);
```
static 상수는 클래스명.상수명 으로 호출하는게 관례입니다